ecosystem/indexer/src/models/token.rs

```diff
@@ -17,11 +17,11 @@ pub struct TokenData {
     pub name: String,
     pub description: String,
     pub max_amount: String,
```

Comment:
similar, please update this to BigDecimal. 
The String here was for a temporary fix.
It is great that this change can support real Number now.